src/utils/trash.sh

```diff
@@ -8,7 +8,7 @@ if [[ -d ~/.local/share/Trash/files ]]; then
 	printf "Should I clean Trash? "
 	read -p "[Y/n]: " ANSW
 	if [[ "$ANSW" == "y" ]]; then
-	    sudo rm -rf ~/.local/share/Trash/*
```

Comment:
Why you deleted that sudo? It's necessary to do a ```rm -rf ``` @archetipico

Reply:
Some systems may not want `sudo`, and if you already inserted your password `rm -rf` should work without `sudo` anyway.

Reply:
> Some systems may not want `sudo`, and if you already inserted your password `rm -rf` should work without `sudo` anyway.

That's right, but you could lose privileges if updates take too much time